src/Behat/Config/Gherkin/Filter/FilterInterface.php

```diff
@@ -0,0 +1,12 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Behat\Config\Gherkin\Filter;
+
+interface FilterInterface
+{
+    public function name(): string;
+
+    public function value(): mixed;
```

Comment:
the interface value returns a mixed but the other current filter return a string.
I'm not aware if some other filters can have other value types, that's why, I've specified mixed on the interface.

Reply:
Right now the types of filters is limited and there is no way to add new filters, so I would say this can be a string

Reply:
done